tests/pipelines/test_pipelines_text_to_audio.py

```diff
@@ -0,0 +1,236 @@
+        speech_generator = pipeline(task="text-to-audio", model="facebook/musicgen-small", framework="pt")
+
+        forward_params = {
+            "do_sample": False,
+            "max_new_tokens": 250,
+        }
+
+        outputs = speech_generator("This is a test", forward_params=forward_params)
+
+        # musicgen sampling_rate is not straightforward to get
+        self.assertIsNone(outputs["sampling_rate"])
+
+        audio = outputs["audio"]
+
+        self.assertEqual(
+            ANY(np.ndarray),
+            audio,
+        )
+
+        # test two examples side-by-side
+        outputs = speech_generator(["This is a test", "This is a second test"], forward_params=forward_params)
+
+        audio = [output["audio"] for output in outputs]
+
+        self.assertEqual(
+            [
+                ANY(np.ndarray),
+                ANY(np.ndarray),
+            ],
+            audio,
+        )
+
+        # test batching
+        outputs = speech_generator(
+            ["This is a test", "This is a second test"], forward_params=forward_params, batch_size=2
+        )
+
+        self.assertEqual(
+            ANY(np.ndarray),
+            outputs[0]["audio"],
+        )
+
+    @slow
+    @require_torch
+    def test_large_model_pt(self):
+        speech_generator = pipeline(task="text-to-audio", model="suno/bark-small", framework="pt")
+
+        # test text-to-speech
+
+        forward_params = {
+            # Using `do_sample=False` to force deterministic output
+            "do_sample": False,
+            "semantic_max_new_tokens": 100,
+        }
+
+        outputs = speech_generator("This is a test", forward_params=forward_params)
+
+        self.assertEqual(
+            {"audio": ANY(np.ndarray), "sampling_rate": 24000},
+            outputs,
+        )
+
+        # test two examples side-by-side
+        outputs = speech_generator(
+            ["This is a test", "This is a second test"],
+            forward_params=forward_params,
+        )
+
+        audio = [output["audio"] for output in outputs]
+
+        self.assertEqual(
+            [
+                ANY(np.ndarray),
+                ANY(np.ndarray),
+            ],
+            audio,
+        )
+
+        # test other generation strategy
+
+        forward_params = {
+            "do_sample": True,
+            "semantic_max_new_tokens": 100,
+            "semantic_num_return_sequences": 2,
+        }
+
+        outputs = speech_generator(
+            "This is a test",
+            forward_params=forward_params,
+        )
+
+        audio = outputs["audio"]
+
+        self.assertEqual(
+            ANY(np.ndarray),
+            audio,
+        )
+
+        # test using a speaker embedding
+
+        processor = AutoProcessor.from_pretrained("suno/bark-small")
+
+        temp_inp = processor("hey, how are you?", voice_preset="v2/en_speaker_5")
+
+        history_prompt = temp_inp["history_prompt"]
+
+        forward_params["history_prompt"] = history_prompt
```

Comment:
Those are too many new lines. Only puts new lines when it makes sense to separate blocks of code please.